spring-boot-project/spring-boot-cli/src/main/java/org/springframework/boot/cli/command/init/ServiceCapabilitiesReportGenerator.java

```diff
@@ -36,8 +36,6 @@
  */
 class ServiceCapabilitiesReportGenerator {
 
-	private static final String NEW_LINE = System.getProperty("line.separator");
```

Comment:
Can we please keep the `NEW_LINE` assignement there so that the PR change is smaller?

Reply:
FWIW, I like the change as @izeye has proposed. I see no need for the constant now that there's a method we can call instead.

Reply:
@snicoll Thanks for the quick feedback! I intentionally removed the constants as I considered `System.lineSeparator()` effectively a constant. What do you think?

Reply:
Thanks for the feedback. Yes, that makes sense.